app/controllers/application_controller.rb

```diff
@@ -93,6 +93,15 @@ def authorize_admin
       current_user.has_role?(Role::ORG_ADMIN, current_organization)
   end
 
+  def require_partner
```

Comment:
Why is this in ApplicationController if it's only used in the partner DashboardController?

Reply:
Future proofing to make it look like the others -- authorize_user, current_role, etc. It looked like those so I put it with those -- wamme to move it?

Reply:
Hmm... I think Partner::BaseController is a fine place to put it. It's still fairly centralized and makes it more obvious that it's only relevant to pages in that namespace.